src/renderer/services/wallet/balances.ts

```diff
@@ -34,29 +35,86 @@ export const reloadBalances: FP.Lazy<void> = () => {
   BCH.reloadBalances()
 }
 
-export const reloadBalancesByChain: (chain: Chain) => FP.Lazy<void> = (chain) => {
+const getServiceByChain = (chain: Chain) => {
```

Comment:
I would not get all service here, just functions we do really need (such as `reloadBalances`, `reloadBalances$`, `resetReload`)

Reply:
yeah. i had implemented it in such way before but there was so much duplication (for reload/reload$/reset there were the same switch/cases) and i've decided to extract it to the separate function. actually... as i can see when we need to implement a new chain we will pass only a new chain-service to the `getServiceByChain` and everything should work as expected.
anyway ^that^ just thoughts. if you wish i will change that to separate switch/cases for every  of reload/reload$/reset getters